client/src/components/client-dashboard-component/FilterCard.jsx

```diff
@@ -0,0 +1,226 @@
+import { useState } from "react";
```

Comment:
We forgot to ask Jake to remove it but Jordan and I decided a while ago to get rid of distinguishing between all closed clients and ones with action needed. Just having the "Closed" checkbox without the additional options should be sufficient here.
<img width="249" alt="Screenshot 2024-01-26 at 12 28 38 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/3a3c6fd6-f27c-4c68-8204-41fd185424d0">